products/nexus.md

```diff
@@ -2,42 +2,279 @@
+    latestReleaseDate: 2023-01-30
+
+-   releaseCycle: "3.45"
+    releaseDate: 2022-12-28
+    support: 2023-12-28
+    eol: 2024-06-28
+    latest: "3.45.1"
+    latestReleaseDate: 2023-01-18
+
+-   releaseCycle: "3.44"
+    releaseDate: 2022-12-15
+    support: 2023-12-15
+    eol: 2024-06-12
+    latest: "3.44.0"
+    latestReleaseDate: 2022-12-15
+
+-   releaseCycle: "3.43"
+    releaseDate: 2022-11-07
+    support: 2023-11-07
+    eol: 2024-05-05
+    latest: "3.43.0"
+    latestReleaseDate: 2022-11-07
+
+-   releaseCycle: "3.42"
+    releaseDate: 2022-09-26
+    support: 2023-09-26
+    eol: 2024-03-24
+    latest: "3.42.0"
+    latestReleaseDate: 2022-09-26
+
+-   releaseCycle: "3.41"
+    releaseDate: 2022-07-27
+    support: 2023-08-03
+    eol: 2024-01-27
+    latest: "3.41.1"
+    latestReleaseDate: 2022-06-24
+
+-   releaseCycle: "3.40"
+    releaseDate: 2022-06-22
+    support: 2023-08-03
+    eol: 2023-12-22
+    latest: "3.40.1"
+    latestReleaseDate: 2022-06-24
+
+-   releaseCycle: "3.39"
+    releaseDate: 2022-05-23
+    support: 2023-08-03
+    eol: 2023-11-19
+    latest: "3.39.0"
+    latestReleaseDate: 2022-05-23
+
+-   releaseCycle: "3.38"
+    releaseDate: 2022-03-03
+    support: 2023-08-03
+    eol: 2023-09-25
+    latest: "3.38.1"
+    latestReleaseDate: 2022-03-29
+
+-   releaseCycle: "3.37"
+    releaseDate: 2021-11-14
+    support: false
+    eol: 2023-05-24
+    latest: "3.37.3"
+    latestReleaseDate: 2021-12-29
+
+-   releaseCycle: "3.36"
+    releaseDate: 2021-10-29
+    support: false
+    eol: 2023-04-29
+    latest: "3.36.0"
+    latestReleaseDate: 2021-10-29
+
+-   releaseCycle: "3.35"
+    releaseDate: 2021-10-13
+    support: false
+    eol: 2023-04-13
+    latest: "3.35.0"
+    latestReleaseDate: 2021-10-13
+
+-   releaseCycle: "3.34"
+    releaseDate: 2021-08-31
+    support: false
+    eol: 2023-03-03
+    latest: "3.34.1"
+    latestReleaseDate: 2021-09-22
+
+-   releaseCycle: "3.33"
+    releaseDate: 2021-08-04
+    support: false
+    eol: 2023-02-04
+    latest: "3.33.1"
+    latestReleaseDate: 2021-08-17
+
+-   releaseCycle: "3.32"
+    releaseDate: 2021-07-06
+    support: false
+    eol: 2023-01-08
+    latest: "3.32.1"
+    latestReleaseDate: 2021-12-20
+
+-   releaseCycle: "3.30"
+    releaseDate: 2021-03-04
+    support: false
+    eol: 2022-09-04
+    latest: "3.30.1"
+    latestReleaseDate: 2021-04-22
+
+-   releaseCycle: "3.29"
+    releaseDate: 2020-12-04
+    support: false
+    eol: 2022-06-04
+    latest: "3.29.2"
+    latestReleaseDate: 2021-01-06
 
 ---
 
 > [Nexus Repository OSS](https://help.sonatype.com/repomanager3/product-information/download) is an open
 > source repository manager developed by Sonatype that supports many artifact formats, including
 > Docker, Java, and npm.
 
-Only the latest version of each major release is supported.
+## [Versioning Scheme](https://help.sonatype.com/docs/sonatype-sunsetting-information/sonatype-nexus-repository-3-versions-status)
```

Comment:
```suggestion
```

No need for a title here. But the link, https://help.sonatype.com/docs/sonatype-sunsetting-information/sonatype-nexus-repository-3-versions-status, is interesting a could be incorporated in the product description.